packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3963,7 +3968,7 @@ export class ContainerRuntime
 			},
 			async (event) => {
 				this.verifyNotClosed();
-				const waitBlobsToAttach = props?.notifyImminentClosure;
+				this.waitBlobsToAttach = props?.notifyImminentClosure;
```

Comment:
this value should never flip flop. we should only set it to true if it is false, otherswise something bad has happend